src/header.rs

```diff
@@ -99,6 +100,15 @@ pub struct GnuSparseHeader {
     pub numbytes: [u8; 12],
 }
 
+#[repr(C)]
+#[allow(missing_docs)]
+pub struct GnuExtSparseHeader {
+    pub sparse: [GnuSparseHeader; 21],
+    pub isextended: [u8; 1],
+}
```

Comment:
Could this also explicitly have the 7 bytes of padding to ensure that the header is 512 bytes in length?